dashboard/app/controllers/courses_controller.rb

```diff
@@ -11,6 +11,7 @@ def index
         @is_english = request.language == 'en'
         @is_signed_out = current_user.nil?
         @force_race_interstitial = params[:forceRaceInterstitial]
+        @header_banner_image_filename = !@is_teacher ? "courses-hero-student" : "courses-hero-teacher"
```

Comment:
Does this mean logged out non-EN get the teacher banner but logged in as a student non-EN gets the student banner?

Reply:
Logged out non-`en` will get the teacher banner only if the URL has `view=teacher`.  Logged in as a student will get the student banner whether they are `en` or no.